README.md

```diff
@@ -14,39 +14,46 @@ rosdep update
 
 # Clone repo
 cd <catkin workspace location>/src
-git clone --recurse-submodules https://github.com/uwrobotics/uwrt_mars_rover.git
+git clone https://github.com/uwrobotics/uwrt_mars_rover.git
 
-# Install all dependencies
-rosinstall --catkin . uwrt_mars_rover/uwrt.rosinstall
+# Install upstream source dependencies
+rosinstall --catkin . uwrt_mars_rover/upstream_dependencies.rosinstall
+
+# Install metapackage source dependencies
+rosinstall --catkin . uwrt_mars_rover/metapackage_dependencies.rosinstall
+
+# Install all binary dependencies
 rosdep install --from-paths . --ignore-src -r -y
 ```
 
```

Comment:
Is there a section on building the workspace, especially what flags to use when building? Something that comes to mind is building the code in release mode so code is more optimized and runs faster. Not sure if any flags are specifically required for the RT_PREEMPT real time functionality.

Reply:
nothing special is needed for rt_prempt. just need to add a syscall to set thread priority

Reply:
for building i have two catkin profiles i personally use. one that's just the default for when i want to iterate code. another has `-Wall -Wextra -Wpedantic -Werror` and also sets flags to run clang-tidy. this matches the ci stages. Could add a config to build in release mode.

Reply:
We could open up a seperate repo for catkin_tools config. or maybe just add the flags to each package's cmakelists? not sure whats a better option.

Reply:
addressed in #45